docs/manage-metadata-using-gravitino.md

```diff
@@ -733,6 +733,159 @@ In addition to the basic settings, Gravitino supports the following features:
 | Bucketed table      | Equal to `CLUSTERED BY` in Apache Hive, some engine may use different words to describe it.  | [Distribution](pathname:///docs/0.3.0/api/java/com/datastrato/gravitino/rel/expressions/distributions/Distribution.html) |
 | Sorted order table  | Equal to `SORTED BY` in Apache Hive, some engine may use different words to describe it.     | [SortOrder](pathname:///docs/0.3.0/api/java/com/datastrato/gravitino/rel/expressions/sorts/SortOrder.html)               |
 
+#### Partitioned table
+
+Currently, Gravitino supports the following partitioning strategies:
+
+:::note
+The `score`, `dt`, and `city` appearing in the table below refer to the field names in a table.
+:::
+
+| Function strategy | Description                                                  | Source types                                                                        | Result type | Json example                                                     | Java example                                    | Equivalent SQL semantics           |
```

Comment:
@qqqttt123 What's your opinion?

Reply:
For a transform, source type and result type are important. Gravitino may not care. But users will care about it.

Reply:
How about adding links to different catalog partitioning docs? 

For the same type and partitioning strategy, it may be feasible in `catalogA` but prohibited in `catalogB`, as this is likely dependent on the catalog's implicit type conversion strategy.

Reply:
OK for me.

Reply:
This may require a few more PRs to refine it, so I will add an issue about it later.